README.md

```diff
@@ -22,64 +21,279 @@
+    ```
+
+    Incorporate complex metadata filters on your queries:
+    ```python
+    from redisvl.query.filter import Tag
+
+    # define a tag match filter
+    tag_filter = Tag("user") == "john"
+
+    # update query definition
+    query.set_filter(tag_filter)
+
+    # execute query
+    results = index.query(query)
+    ```
+
+- [RangeQuery](https://www.redisvl.com/api/query.html#rangequery) - Vector search within a defined range paired with customizable filters
+- [FilterQuery](https://www.redisvl.com/api/query.html#filterquery) - Standard search using filters and the full-text search
+- [CountQuery](https://www.redisvl.com/api/query.html#countquery) - Count the number of indexed records given attributes
+
+> Read more about building advanced queries [here](https://www.redisvl.com/user_guide/hybrid_queries_02.html).
+
+
+### 🖥️ Command Line Interface
+Create, destroy, and manage Redis index configurations from a purpose-built CLI interface: `rvl`.
 
 ```bash
-pip install redisvl
+$ rvl -h
+
+usage: rvl <command> [<args>]
+
+Commands:
+        index       Index manipulation (create, delete, etc.)
+        version     Obtain the version of RedisVL
+        stats       Obtain statistics about an index
 ```
 
-For more instructions, see the [installation guide](https://www.redisvl.com/overview/installation.html).
+> Read more about using the `redisvl` CLI [here](https://www.redisvl.com/user_guide/cli.html).
+
+### ⚡  Community Integrations
+Integrate with popular embedding models and providers to greatly simplify the process of vectorizing unstructured data for your index and queries:
+- [Cohere](https://www.redisvl.com/api/vectorizer/html#coheretextvectorizer)
+- [OpenAI](https://www.redisvl.com/api/vectorizer.html#openaitextvectorizer)
+- [HuggingFace](https://www.redisvl.com/api/vectorizer.html#hftextvectorizer)
+- [GCP VertexAI](https://www.redisvl.com/api/vectorizer.html#vertexaitextvectorizer)
 
-## Getting Started
+```python
+from redisvl.utils.vectorize import CohereTextVectorizer
 
-To get started with RedisVL, check out the
+# set COHERE_API_KEY in your environment
+co = CohereTextVectorizer()
+
+embedding = co.embed(
+    text="What is the capital city of France?",
+    input_type="search_query"
+)
+
+embeddings = co.embed_many(
+    texts=["my document chunk content", "my other document chunk content"],
+    search_type="search_documents"
+)
+```
+
+> Learn more about using `redisvl` Vectorizers in your workflows [here](https://www.redisvl.com/user_guide/vectorizers_04.html).
+
+### 💫 Beyond Vector Search
+Modern GenAI applications require much more than RAG-style vector search in order
+to perform well in production. `redisvl` provides some common extensions that
+aim to improve applications working with LLMs:
+
+- **LLM Semantic Caching** is designed to increase the request QPS, reduce the cost of using LLM models in production, and drive towards more compliant + consistent responses, robust to nuanced input.
```

Comment:
I'm finding this sentence a tough read, specifically the ending "robust to nuanced input"